tests/src/Interop/common/xplatform.h

```diff
@@ -57,7 +57,7 @@
 #define WINAPI __stdcall
 #endif
```

Comment:
Hmm , I thought WINAPI was defined to be _cdecl on non-windows platform  , obviously i am wrong.
LGTM